src/js/themes/hpe.js

```diff
@@ -211,57 +207,26 @@ export const hpe = deepFreeze({
   },
   button: {
     'cta-primary': {
-      background: {
-        color: 'brand',
-      },
-      border: undefined,
-      color: 'white', // 'text-strong',
-      font: {
-        weight: 700,
-      },
-      // TODO: Enhance Grommet to allow theme to set `button.{kind}.icon` which is an SVG.
-      // With this enhancement, the `extend` property definition should be able to be removed.
-      extend: (props) => {
-        const color = props.disabled ? 'text-xweak' : 'text-strong';
-        const dark = props.active || props.disabled ? props.theme.dark : true;
-        const colorValue =
-          props.theme.global.colors[color][dark ? 'dark' : 'light'];
-
-        return `&:after {
-            display: inline-block;
-            width: 48px;
-            height: 20px;
-            padding-left: ${props.hasLabel ? '12px' : '0px'};
-            vertical-align: text-bottom;
-            content: ${hpeElement(colorValue)};
-          }`;
+      background: { color: 'brand' },
+      border: {
+        radius: '100px',
       },
+      color: 'white', // text-strong
+      font: { weight: 'bold' },
+      icon: <Hpe />,
+      reverse: true,
     },
     'cta-alternate': {
-      background: {
-        color: 'background-contrast',
+      background: 'background-contrast',
```

Comment:
What can we do to make this appear appropriately on dark backgrounds?

I think we may need to get an explicit grey instead of something with opacity.

Reply:
Fixed in latest commit.